src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/OceanEcsCluster.java

```diff
@@ -0,0 +1,252 @@
+package com.spotinst.sdkjava.model.bl.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Date;
+import java.util.Set;
+import java.util.HashSet;
+
+/**
+ * Created by Bansi Chapla on 20/07/2021.
+ */
+
+public class OceanEcsCluster {
+
+    @JsonIgnore
+    private Set<String>                            isSet;
+    private String                                 id;
+    private String                                 name;
+    private String                                 region;
+    private String                                 clusterName;
+    private ClusterAutoScalerConfiguration         autoScaler;
+    private ClusterCapacityConfiguration           capacity;
+    private ClusterComputeConfiguration            compute;
+    private ClusterSchedulingConfiguration         scheduling;
+    private ClusterStrategyConfiguration           strategy;
+    private Date                                   createdAt;
```

Comment:
Do we need 'createdAt' and 'updatedAt' fields.

Reply:
Not needed. Tried to Aligned with Elastigroup creation Example. 
Removed CreatedAt and UpdatedAt fields.